packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -109,62 +115,51 @@ export default function ListViewSidebar( { listViewToggleElement } ) {
 		handleToggleListViewShortcut
 	);
 
-	/**
-	 * Render tab content for a given tab name.
-	 *
-	 * @param {string} tabName The name of the tab to render.
-	 */
-	function renderTabContent( tabName ) {
-		if ( tabName === 'list-view' ) {
-			return (
-				<div className="edit-post-editor__list-view-panel-content">
-					<ListView dropZoneElement={ dropZoneElement } />
-				</div>
-			);
-		}
-		return <ListViewOutline />;
-	}
-
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
 		<div
 			className="edit-post-editor__document-overview-panel"
 			onKeyDown={ closeOnEscape }
 			ref={ sidebarRef }
 		>
-			<Button
-				className="edit-post-editor__document-overview-panel__close-button"
-				icon={ closeSmall }
-				label={ __( 'Close' ) }
-				onClick={ closeListView }
-			/>
```

Comment:
As @andrewhayward pointed out, `Button` used to come before the tablist in DOM order and therefore in tab order too. 

Can we move it back to before the tablist? If necessary, we should be able to play around with CSS and DOM structure to write a cleaner layout (ie. no position absolute for the close button), thanks to the flexibility that `Tabs` offers with its subcomponents

Reply:
@andrewhayward  & @ciampo Thank you both for the input regarding the order of the elements and tab flow! I've moved it back before the `tablist`, and updated the related styles in a454774478 so we're no longer relying on `position: absolute`. I did end up adding a wrapper `div` which I'd have preferred to avoid, but having a `flex` container made life a lot easier.